test/req-stat-stdin.test

```diff
@@ -7,7 +7,7 @@ if [ "." != "$TEST_ORIGIN" ]; then
 	cp $TEST_ORIGIN/SPEC.docx .
 	cp $TEST_ORIGIN/TEST.txt .
 fi
-$REQFLOW stat -s -c - > $T.stdout.out 2> $T.stderr.out << EOF
+$(! $REQFLOW stat -s -c - > $T.stdout.out 2> $T.stderr.out << EOF
```

Comment:
The $(...) makes the code harder to read.
Please use test_run, that I introduced late during the coding, and is intended for encapsulating new tests (and having a common way of coding the tests).

Like this :

```
 . ./envtest
+. $srcdir/functions
 T=req-stat-stdin

...

-$REQFLOW stat -s -c - > $T.stdout.out 2> $T.stderr.out << EOF
+test_run $REQFLOW stat -s -c - << EOF

...

-diff -w $T.stdout.out $TEST_ORIGIN/$T.stdout.ref
-diff -w $T.stderr.out $TEST_ORIGIN/$T.stderr.ref
+diff -w $T.stdout $TEST_ORIGIN/$T.stdout.ref
+diff -w $T.stderr $TEST_ORIGIN/$T.stderr.ref

```

Reply:
Done in the last commit.